doc/user/sql/create-sink.md

```diff
@@ -75,13 +81,78 @@ CREATE MATERIALIZED VIEW frank_quotes AS
     WHERE attributed_to = 'Frank McSherry';
 ```
 ```sql
-CREATE SINK quotes_sink
+CREATE SINK frank_quotes_sink
 FROM frank_quotes
 INTO KAFKA BROKER 'localhost' TOPIC 'frank-quotes-sink'
 FORMAT AVRO USING
     CONFLUENT SCHEMA REGISTRY 'http://localhost:8081';
 ```
 
+### Get actual topic names
+
+```sql
+SELECT * FROM mz_catalog_names NATURAL JOIN mz_kafka_sinks;
+```
+
+```nofmt
+ global_id |              name                    |                        topic
+-----------+--------------------------------------+------------------------------------------------------
+ u5        | materialize.public.quotes_sink       | quotes-sink-u6-1586024632-15401700525642547992
+ u6        | materialize.public.frank_quotes_sink | frank-quotes-sink-u5-1586024632-15401700525642547992
+```
+
+### Avro OCF sinks
+
+When creating Avro Object Container File (OCF) sinks, Materialize creates a new sink file using the naming scheme defined above and appends the Avro schema data in its header.
+
+## Examples
```

Comment:
Can we restructure the examples so they don't collide? I have a slight preference for each doc to have one `Examples` section that's an `h2`, and then develop some other taxonomy from there, e.g. `### Kafka sinks`, `### Avro OCF sinks` and then make each example an `h4`. If you dislike that, can we amend the `Examples` text to make clear which type they're an example of?

Reply:
done I like that format better too!